### PR TITLE
chore (demo app) : restructure tab layout

### PR DESCRIPTION
**Pull request by rohanbayya**
asana: https://app.asana.com/0/1204008699308084/1208057868583172/f

restructure tab layout
navigator & navigation -> navigation tab
custom components, case studies -> advanced tab

### CODE REVIEW

File: examples/_includes/macros/tab/styles.xml.njk [hunk 1]
```diff
@@ -1,2 +1,35 @@
...
+  borderColor="transparent"
+  borderRadius="8"
+  borderWidth="1"
+  flex="1"
+  padding="8"
+  justifyContent="center"
+>
+  <modifier selected="true">
+  <style
+  backgroundColor="white"
```
Comment (flochtililoch) [inline examples/_includes/macros/tab/styles.xml.njk:25]: is prettier causing this indentation issue?

File: examples/_includes/macros/tab/styles.xml.njk [hunk 2]
```diff
@@ -1,2 +1,35 @@
 <style id="tab-content" flex="1" />
 <style id="centered" alignItems="center" justifyContent="center" />
+<style
+  id="row"
```
Comment (flochtililoch) [inline examples/_includes/macros/tab/styles.xml.njk:4]: we should use a prefix for all the styles so there's no risk of collision with other styles
```suggestion
  id="tab-row"
```

File: examples/advanced/case-studies/basic-forms/index.xml.njk [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-permalink: "/case-studies/basic-forms/index.xml"
+permalink: "/advanced/case-studies/basic-forms/index.xml"
```
Comment (flochtililoch) [inline examples/advanced/case-studies/basic-forms/index.xml.njk:2]: given that we now have a middleware that generates the permalink, should we plan to remove them all from every files?

**Review by flochtililoch — COMMENTED** [review]
it would be worth adding 2 before/after screenshots to the PR description to show what's changed in the UI.

File: examples/_includes/macros/tab/styles.xml.njk (hunk 1 above)
Comment (rohanbayya) [reply examples/_includes/macros/tab/styles.xml.njk:25]: yes ;/

File: examples/advanced/case-studies/basic-forms/index.xml.njk (hunk 3 above)
Comment (rohanbayya) [reply examples/advanced/case-studies/basic-forms/index.xml.njk:2]: yes, will handle this in another PR

**Review by flochtililoch — APPROVED** [review]

**Pull request merged**